v4/source/channels.yaml

```diff
@@ -189,6 +189,42 @@
             // GetPublicChannelsByIdsForTeam
             channels, resp := Client.GetPublicChannelsByIdsForTeam(<TEAMID>, channelIds)
 
+  /channels/{channel_id}/timezones:
+    get:
+      tags:
+        - channels
+      summary: Get users timezones for a channel
```

Comment:
Maybe update this to `Get timezones in a channel`

Reply:
done